src/Bridge/Psr7/RequestFactory.php

```diff
@@ -95,6 +95,23 @@ public static function fromLambdaEvent(array $event): ServerRequestInterface
             'REQUEST_URI' => $uri,
         ];
 
+        if (isset($event['requestContext'])
+            && array_key_exists('identity', $event['requestContext'])
+            && array_key_exists('sourceIp', $event['requestContext']['identity'])) {
+            $server['REMOTE_ADDR'] = $event['requestContext']['identity']['sourceIp'];
+        }
+
+        if (isset($headers['X-Forwarded-For'])) {
+            $ips = [$headers['X-Forwarded-For']];
+            if (false !== strpos($ips[0], ',')) {
+                $ips = preg_split('/,\s?/', $ips[0]);
+                $ip = $ips[count($ips) - 1];
+                if (\filter_var($ip, FILTER_VALIDATE_IP)) {
+                    $server['REMOTE_ADDR'] = $ip;
```

Comment:
re-assignment of the ip should only be based on a ip whitelist

Reply:
I see your point, as I intend to use this along with a trusted proxy list. When using symfony, whitelisting doesn't work without a `REMOTE_ADDR`. Perhaps you have another suggestion to better mesh the `REMOTE_ADDR` property? In truth, aren't all IPs fairly spoofable? With that in mind, I'm just looking to get the initial values passed into symfony, then I'll whitelist.